src/agentscope/service/__init__.py

```diff
@@ -26,6 +26,12 @@
     dblp_search_authors,
     dblp_search_venues,
 )
+from .reasoning.wolfram_alpha import (
+    query_wolfram_alpha_short_answers,
```

Comment:
Please rename all services into `wolfram_alpha_xxx_search` or `wolfram_alpha_xxx_query` to keep consistent with the existing services like `openai_image_to_text` and `bing_search`

Reply:
Updated according to the comment.